src/rules/no-obsolete-element/index.js

```diff
@@ -0,0 +1,58 @@
+import { utils } from 'stylelint';
+import isStandardSyntaxRule from 'stylelint/lib/utils/isStandardSyntaxRule';
+import { obsoleteElements } from './obsoleteElements';
+
+export const ruleName = 'a11y/no-obsolete-element';
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: selector => `Unexpected using obsolete selector "${selector}"`,
+});
+
+function check(selector, node) {
+  if (node.type !== 'rule') {
+    return true;
+  }
+  if (obsoleteElements.has(selector)) {
```

Comment:
May I make an advice?

I guess, you may rewrite this code to:
```js
return !obsoleteElements.has(selector)
```

What do you think?

Reply:
Sure, I'll change it